code/modules/cargo/packs/armory.dm

```diff
@@ -216,23 +205,6 @@
 					/obj/item/clothing/gloves/combat)
 	crate_name = "swat crate"
 
-/datum/supply_pack/security/armory/swattasers //Lesser AEG tbh
```

Comment:
SWAT Tasers are hos gun + 1-2 more shots.
People rarely get them do to AEGs being better from an ammo to shot converation. (2 AEG = 1 SWAT taser ammo wise, Not counting the auto charge.)

Reply:
By the by the commit "Lesser AEG tbh" is do to them not being able to auto charge. But have somewhat the sameish ammo.